server/fishtest/templates/users.mak

```diff
@@ -15,48 +15,66 @@
+    <div class="card card-lg-sm text-center">
+      <div class="card-header text-nowrap" title="Testers">Testers</div>
+      <div class="card-body">
+        <h4 class="card-title mb-0 monospace" style="font-size: 20px;">
+          ${sum(u['str_last_updated'] != 'Never' for u in users)}
+        </h4>
       </div>
     </div>
-    <div class="row">
-      <div class="col text-end"><b>Developers</b></div>
-      <div class="col text-start">
-        ${sum(u['tests'] > 0 for u in users)}
+  </div>
+  <div class="col-6 col-sm">
+    <div class="card card-lg-sm text-center">
+      <div class="card-header text-nowrap" title="Developers">Developers</div>
+      <div class="card-body">
+        <h4 class="card-title mb-0 monospace" style="font-size: 20px;">
+          ${sum(u['tests'] > 0 for u in users)}
+        </h4>
       </div>
     </div>
   </div>
 
-  <div class="col-sm">
-    <div class="row">
-      <div class="col text-end"><b>Active testers</b></div>
-      <div class="col text-start">
-        ${sum(u['games_per_hour'] > 0 for u in users)}
+  <div class="col-6 col-sm">
+    <div class="card card-lg-sm text-center">
+      <div class="card-header text-nowrap" title="Active testers">Active testers</div>
+      <div class="card-body">
+        <h4 class="card-title mb-0 monospace" style="font-size: 20px;">
+          ${sum(u['games_per_hour'] > 0 for u in users)}
+        </h4>
       </div>
     </div>
-    <div class="row">
-      <div class="col text-end"><b>Tests submitted</b></div>
-      <div class="col text-start">
-        ${sum(u['tests'] for u in users)}
+  </div>
+  <div class="col-6 col-sm">
+    <div class="card card-lg-sm text-center">
+      <div class="card-header text-nowrap" title="Tests submitted">Tests submitted</div>
+      <div class="card-body">
+        <h4 class="card-title mb-0 monospace" style="font-size: 20px;">
+          ${sum(u['tests'] for u in users)}  
+        </h4>
       </div>
     </div>
   </div>
 
-  <div class="col-sm">
-    <div class="row">
-      <div class="col text-end"><b>Games played</b></div>
-      <div class="col text-start">
-        ${sum(u['games'] for u in users)}
+  <div class="col-6 col-sm">
+    <div class="card card-lg-sm text-center">
+      <div class="card-header text-nowrap" title="Games played">Games played</div>
+      <div class="card-body">
+        <h4 class="card-title mb-0 monospace" style="font-size: 20px;">
+          ${sum(u['games'] for u in users)}
+        </h4>
       </div>
     </div>
-    <div class="row">
-      <div class="col text-end"><b>CPU time</b></div>
-      <div class="col text-start">
-        ${f"{sum(u['cpu_hours'] for u in users)/(24*365):.2f} years"}
+  </div>
+  <div class="col-6 col-sm">
+    <div class="card card-lg-sm text-center">
+      <div class="card-header text-nowrap" title="CPU time">CPU time (years)</div>
+      <div class="card-body">
+        <h4 class="card-title mb-0 monospace" style="font-size: 20px;">
```

Comment:
With the "years" removed u can also try removing the hardcoded font-size

Reply:
well, that's intended but as you wish.